bin/foreman-installer

```diff
@@ -2,11 +2,7 @@
 require 'rubygems'
 require 'kafo'
 
-if Dir.glob('config/*.yaml').any?
-  CONFIG_DIR = 'config/'
-else
-  CONFIG_DIR = '/etc/foreman-installer/scenarios.d/'
```

Comment:
Yes, I changed the Rakefile too at https://github.com/theforeman/foreman-installer/pull/173/files/6fef25e5d3a41f46bff9794946b2c6faed6c84f5#diff-52c976fc38ed2b4e3b1192f8a8e24cffR34 to always replace this CONFIG_DIR line with the path it's installing to.

The packaged version used to still contain this conditional, so it would have:

<pre>
if Dir.glob('config/*.yaml').any?
  CONFIG_DIR = 'config/'
else
  CONFIG_DIR = '/etc/foreman-installer/scenarios.d/'
end
</pre>

Now it'll be a single `CONFIG_DIR = '/etc/foreman-installer/scenarios.d'` generated by the Rakefile.

I don't think there's a good reason to keep the config/*.yaml support in the packaged/installed version, and I've seen reports of it causing problems when config/*.yaml does exist.